src/sign-in-with-google/sign-in-with-google.php

```diff
@@ -15,7 +15,7 @@
  * Plugin Name:       Sign In With Google
  * Plugin URI:        http://www.northstarmarketing.com
  * Description:       Adds a "Sign in with Google" button to the login page, and allows users to sign up and login using Google.
- * Version:           1.3.0
+ * Version:           1.3.1
```

Comment:
This seems more like a feature to me, wouldn't it make more sense for this to be version 1.4?

Reply:
Yeah, `1.4.0` seems like a better way to represent the change. 👍